src/typechecker.erl

```diff
@@ -4941,6 +4941,7 @@ type_check_forms(Forms, Opts) ->
     AllErrors = lists:foldr(fun (Function, Errors) ->
                                     type_check_form_with_timeout(Function, Errors, StopOnFirstError, Env, Opts)
                             end, [], ParseData#parsedata.functions),
+    timer:sleep(200),
```

Comment:
:scream:

Reply:
This is a draft PR and this lines comes from a commit title "debug" ;)

Reply:
For some context, if we trace execution (as set up in `gradualizer_app`), we need this sleep, as otherwise the VM exits before the tracer process manages to print all the messages we want to see.